### PR TITLE
WIP: Review and Submit page

### PR DESCRIPTION
**Pull request by tatyree**
Do not merge

Closes https://github.com/JudicialAppointmentsCommission/digital-platform/issues/59

### CODE REVIEW

File: src/views/FormSections/PersonalDetails/Show.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        </tr>
+      </tbody>
+    </table>
+  </section>
+</template>
+
+<script>
+
+export default {
+  name: "Personal",
```
Comment (ollietreend) [inline src/views/FormSections/PersonalDetails/Show.vue]: Let's remove `name` keys on component where we see them – we don't need these, they're just cruft.

File: src/views/FormSections/PersonalDetails/Show.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  data() {
+    return {
+      applicant: this.$store.getters.applicant(),
+      isSaving: false,
```
Comment (ollietreend) [inline src/views/FormSections/PersonalDetails/Show.vue]: The `isSaving` property isn't used in the read-only components – please remove.

File: src/views/FormSections/ReviewAndSubmit.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import Qualifications from '@/views/FormSections/Qualifications/Show';
+export default {
+  name: "ReviewAndSubmit",
+  components: {
+    Personal,
+    Qualifications
+  },
+  data() {
+    return {
+      vacancy: this.$store.getters.vacancy,
```
Comment (ollietreend) [inline src/views/FormSections/ReviewAndSubmit.vue]: Looks like `vacancy` property isn't needed here.

**Comment by tatyree** [comment]
@ollietreend made some breaking changes that make this branch untenable as a whole.  I'm closing the PR and will cherry pick the relevant changes out of it once I've done some groundwork on the app structure.

**Pull request closed without merging**